### ISSUE TITLE
Feature Request: Open link in Reader View

### ISSUE THREAD

**Issue by mortyobnoxious** [Robbendebiene/Gesturefy#351]
Hi, latest UI is great.

I just wanted to ask if it's possible to add two more actions:
● Open link in [Sidebar](https://testpilot.firefox.com/experiments/side-view/)
● Open link in [Reader View](https://addons.mozilla.org/en-US/firefox/addon/reader-view/)

That would be so useful.

**Comment by Robbendebiene** [comment]
Hey, I'm glad you like the new UI.
"Open link in Sidebar" is only possible if the provided add-on supports its own API (which it currently does not)
"Open link in Reader View" should be possible, but not every link/website can be displayed in reader view. So maybe a command/action that allows you to toggle reader view for the current website would make more sense?

**Comment by mortyobnoxious** [comment]
> toggle reader view for the current website would make more sense

That would work but if I'm not mistaken, Firefox already has such a thing in address bar. 

What I meant is while browsing some websites such as Reddit and Twitter which contains lots of external links, opening links in Reader View in new tab directly so that one can instantly read articles and such without being have to experience horrific website designs.

Of course, while opening links, user will have to decide that the target website is article based or not.

**Comment by Robbendebiene** [comment]
We've discussed about this feature and came to the conclusion that this could be an extra option for the "Open Link ..." commands. However we do not want to clutter the add-on with too many specific options wherefore we are currently unsure if we will add this option or not.

An alternative we've found would be the [automatic reader view](https://addons.mozilla.org/en-US/firefox/addon/automatic-reader-view/) extension, which allows you to always enable reader mode for specific websites. Depending on your needs this might be an even better solution and we wouldn't have to add an extra option :)

If this does not satisfy your desires, then there is still the question about how the "open site in reader mode" command will behave.
Basically there are two posibilities:
1. Force reader mode: This means that the website will be loaded immediately in reader mode. The drawback of this is that Firefox reader mode will display an error if the website cannot be displayed in reader mode.
2. Toggle reader mode if possible: This means that the website will be loaded normally and the reader mode will be enabled afterwards if possible. The drawback here is that you will always see the normal website first for a short time.

**Comment by mortyobnoxious** [comment]
> Force reader mode: This means that the website will be loaded immediately in reader mode. The drawback of this is that Firefox reader mode will display an error if the website cannot be displayed in reader mode.

> this could be an extra option for the "Open Link ..." commands. 

That would be great, if it's possible. Basically it will **open link in a new tab**, but with the prefix "_about:reader?url=_".

For example when I apply a gesture on:
https://support.mozilla.org/en-US/questions/1188877

It will be opened like this whether the website support Reader Mode or not:
about:reader?url=https://support.mozilla.org/en-US/questions/1188877

Thank you for your consideration.

**Comment by Robbendebiene** [comment]
A command has been added to Toggle the reader mode with the release of [Gesturefy version 3.0](https://github.com/Robbendebiene/Gesturefy/releases/tag/v3.0.0).
